### ISSUE TITLE
[Design] - Profile: update alert audit documentation

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#30458]
Update [FE docs](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/maintenance/front-end/alert-updates-Sept-2021.md) to include

- [x] Numbering for each use case
- [x] Cleaned up formatting
- [x] Whether we're making an update or adding something new

**Comment by andaleliz** [comment]
@ajakabcin fyi created this ticket for this sprint

**Comment by andaleliz** [comment]
@ajakabcin, @erikphansen, and @tmitchellgcio, the docs have been updated. I changed the organization from organized by use case, to organized by New, Updates, or No Changes.  The alerts in each section are numbered.

I also added a couple of other values to each one in hopes that it's easier to create the FE tickets and get a sense of what's going on:
- Impacted sections
- what's changing (for alert updates)

Lmk if you have questions
